docs/search_config.md

```diff
@@ -0,0 +1,19 @@
+# Search Algorithm Configuration
+
+The calibration process utilizes a genetic algorithm. The configuration file includes parameters that allow users to tune the genetic algorithm for optimal performance.
+
+- **population_size**: Defines the number of candidate solutions in each generation. A larger population increases diversity and enhances the search, but require more computational resources. Very small populations may lead to premature convergence.
+- **generations**: Specifies the maximum number of iterations the algorithm can perform. Increasing this value allows more opportunities for improvement, but leads to longer runtimes. The calibration process will terminate early if an optimal solution is found before reaching this limit.
+- **mutation_probability**: Determines the likelihood of random changes being introduced in offspring. Higher mutation rates encourage exploration of new solutions but may destabilize good candidates.
+- **crossover_probability**: Specifies the probability that two parent solutions will combine to produce a child. Higher crossover rates encourage the exploitation of good solutions; however, excessive rates may reduce overall diversity.
+
+## Optimizing Parameters: Practical Guidance
+
+- **Testing with Small Values:** Employ reduced population sizes and fewer generations during initial testing phases to quickly validate code and workflow.
+- **Calibration with Larger Values:** For full-scale calibration, increase both population_size and generations to achieve a more comprehensive search. Studies recommend population sizes between 30–100 and similar ranges for generations, particularly for building energy applications.
```

Comment:
I like that you recommend specific ranges here. Can we do that for more of the parameters?

Can we either link to the studies that support your statement, or instead of `Studies` say `We`?

Reply:
Should we mention that there is a relationship between the population_size/generations inputs and the number/granularity of calibration adjustment knobs? If you don't have many knobs available, I assume a large population_size or number of generations will be nonsensical.

Reply:
> I like that you recommend specific ranges here. Can we do that for more of the parameters?
> 
> Can we either link to the studies that support your statement, or instead of `Studies` say `We`?

Actually, while I do have papers that I support this recommendation, the ranges they used were specific to their studies. They are not claiming that these values must be used. That’s why I think phrasing it as “we recommend” makes more sense. The same applies to other parameters as well. Rather than giving very specific ranges, I describe them simply as high/low for this reason. Also, since it’s often necessary to experiment with different values to find the optimal settings for a particular problem, providing too definite a range could be risky.